Framework/LiveData/inc/MantidLiveData/Kafka/KafkaHistoListener.h

```diff
@@ -0,0 +1,64 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2008 ISIS Rutherford Appleton Laboratory UKRI,
+//     NScD Oak Ridge National Laboratory, European Spallation Source
+//     & Institut Laue - Langevin
+// SPDX - License - Identifier: GPL - 3.0 +
+#ifndef MANTID_LIVEDATA_ISISKAFKAHISTOLISTENER_H_
+#define MANTID_LIVEDATA_ISISKAFKAHISTOLISTENER_H_
+//----------------------------------------------------------------------
+// Includes
+//----------------------------------------------------------------------
+#include "MantidAPI/LiveListener.h"
+
+//----------------------------------------------------------------------
+// Forward declarations
+//----------------------------------------------------------------------
+
+namespace Mantid {
+namespace LiveData {
+class KafkaHistoStreamDecoder;
+
+/**
+  Implementation of a live listener to consume messages from the Kafka system
```

Comment:
I think this comment is a copy-paste error @LamarMoore @MikeHart85

Reply:
Yes this, like much of this code, was copied from:

* [KafkaEventListener.h](https://github.com/mantidproject/mantid/blob/fcc35cce4241f26f627100f008655ebe7036720b/Framework/LiveData/inc/MantidLiveData/Kafka/KafkaEventListener.h)
* [KafkaEventListener.cpp](https://github.com/mantidproject/mantid/blob/fcc35cce4241f26f627100f008655ebe7036720b/Framework/LiveData/src/Kafka/KafkaEventListener.cpp)
* [KafkaEventStreamDecoder.h](https://github.com/mantidproject/mantid/blob/fcc35cce4241f26f627100f008655ebe7036720b/Framework/LiveData/inc/MantidLiveData/Kafka/KafkaEventStreamDecoder.h)
* [KafkaEventStreamDecoder.cpp](https://github.com/mantidproject/mantid/blob/fcc35cce4241f26f627100f008655ebe7036720b/Framework/LiveData/src/Kafka/KafkaEventStreamDecoder.cpp)

This comment should be updated to reflect that this listener works with histograms rather than events.